docs/install.md

```diff
@@ -985,13 +985,80 @@ steps needed to configure SAT to use externally-accessible API endpoints exposed
    $ . ${SAT_VENV_PATH}/bin/activate
    ```
 
-1. Install the SAT Python package in the virtual environment.
+1. Clone the SAT source code.
+
+   **Note:** To use SAT version 3.19, this example clones the `release/3.19` branch of
+   `Cray-HPE/sat`. However, for better clarity, these instructions include steps that apply only to
+   versions newer than 3.19. Specifically, the instructions include references to the
+   `csm-api-client` package, which was not a dependency of SAT in version 3.19.
+
+   ```screen
+   (venv) $ git clone --branch=release/3.19 https://github.com/Cray-HPE/sat.git
+   ```
+
+1. Set up the SAT CSM Python dependencies to be installed from their source code.
+
+   SAT CSM Python dependency packages are not currently distributed publicly as
+   source packages or binary distributions. They must be installed from
+   their source code hosted on GitHub. Also, to install the `cray-product-catalog`
+   Python package, you must first clone it locally. Use the following steps to
+   modify the SAT CSM Python dependencies so they can be installed from their source code.
+
+   1. Clone the source code for `cray-product-catalog`.
+
+      ```screen
+      (venv) $ git clone --branch v1.6.0 https://github.com/Cray-HPE/cray-product-catalog
+      ```
+
+   1. In the `cray-product-catalog` directory, create a file named `.version`
+      that contains the version of `cray-product-catalog`.
+
+      ```screen
+      (venv) $ echo 1.6.0 > cray-product-catalog/.version
+      ```
+
+   1. Open the "locked" requirements file in a text editor.
+
+      ```screen
+      (venv) $ vim sat/requirements.lock.txt
+      ```
+
+   1. Update the line containing `cray-product-catalog` so that it reflects the local path
+      to `cray-product-catalog`.
+
+      It should read as follows:
+
+      ```screen
+      ./cray-product-catalog
+      ```
+
+   1. For versions of SAT newer than 3.19, change the line containing `csm-api-client` to
+      read as follows:
+
+      ```screen
+      csm-api-client@git+https://github.com/Cray-HPE/python-csm-api-client@release/1.1
+      ```
+
+   1. (Optional) Confirm that `requirements.lock.txt` is modified as expected.
+
+      **Note:** If installing a version newer than 3.19, you should only see `csm-api-client`.
```

Comment:
I think this note is wrong. I think if installing 3.19, you will only see `cray-product-catalog`. If installing newer than 3.19, you will see both.

Perhaps the "only" is just in the wrong place. It would make sense if the sentence stated "Only if installing a version newer than 3.19...."

Reply:
Oh yeah, I see that it was correct in the first version and then was misinterpreted and changed in an earlier review. Perhaps my rewording in the first paragraph is more clear.

Reply:
> Perhaps the "only" is just in the wrong place. It would make sense if the sentence stated "Only if installing a version newer than 3.19...."

Oops, yes that is correct. It originally said "You should only see `csm-api-client` if installing a version greater than 3.19," which was ambiguous.

Reply:
I pushed a commit addressing this comment. I still think it's a little awkward sounding if you or @jld-dehn-hpe have further ideas for rewording it. I wanted to make it really unambiguous that "only" is referring to "only if you are installing >3.19" not "only `csm-api-client`".

Reply:
@ekamin-hpe, I suggested a different way of wording this. You'll have to see if you like it better.